TrkPatRec/fcl/prolog.fcl

```diff
@@ -150,13 +150,50 @@ TimeClusterFinderUpi : {
 RobustHelixFinder : {
     module_type		: RobustHelixFinder
     ComboHitCollection     : "makePH"
-    ComboHitFlagCollection : "FlagBkgHits:ComboHits"
     HelixStereoHitMVA      : { MVAWeights : "TrkPatRec/test/HelixStereoHitMVA.weights.xml" }
     HelixNonStereoHitMVA   : { MVAWeights : "TrkPatRec/test/HelixNonStereoHitMVA.weights.xml" }
-    diagPlugin : { tool_type                    : "RobustHelixFinderDiag"
-	mcTruth                                 : 0
+    DiagPlugin : { 
+	diagLevel                    : 0
+	tool_type                    : "RobustHelixFinderDiag"
+	mcTruth                      : 0
     }
+    TimeClusterCollection : "TTtimeClusterFinder"
```

Comment:
Why is this looking for trigger collection?  i would expect to see the reco collection here, and TT* in prolog_trigger.fcl

Reply:
Good catch! This Isa typo

Reply:
Fixed!